jlog.c

```diff
@@ -2179,6 +2179,139 @@ int jlog_ctx_read_message(jlog_ctx *ctx, const jlog_id *id, jlog_message *m) {
   }
   return -1;
 }
+int jlog_ctx_bulk_read_messages(jlog_ctx *ctx, const jlog_id *id, const int count, jlog_message *m) {
```

Comment:
I'd rather not change the existing path; I'm hesitant to change existing functionality.